storey/drivers.py

```diff
@@ -244,7 +246,6 @@ def _build_feature_store_update_expression(self, aggregation_element, additional
                     expressions.append(f'{name}={self._convert_python_obj_to_expression_value(value)}')
                 else:
                     expressions.append(f'REMOVE {name}')
-
```

Comment:
No need to change formatting unnecessarily.

Reply:
fixed